app/controllers/admin/action_pages_controller.rb

```diff
@@ -36,12 +38,17 @@ def new
     @actionPage.email_campaign = EmailCampaign.new
     @actionPage.congress_message_campaign = CongressMessageCampaign.new
     @actionPage.email_text = Rails.application.config.action_pages_email_text
+    5.times { @actionPage.affiliation_types.build }
   end
 
   def create
     @actionPage = ActionPage.new(action_page_params.merge(author: current_user))
 
     if @actionPage.save
+      if institutions_params[:category]
```

Comment:
This condition should trigger `ActionInstitution.delay.add` in the update action as well. That said, it looks like it may make more sense to add a `institution_category` column to action pages, and use that as the foreign key for institutions? Delaying the import here also seems to create a race condition when the page reloads, due to this line

```ruby
@target_category = @actionPage.institutions.first.category
```